design/python-api.md

```diff
@@ -0,0 +1,84 @@
+# Python REST API
+
+The Python REST API design is as follows:
+
+##  experiment_trials
+
+-   Start or continue an experiment.
+    
+    `POST /experiment_trials`
+    
+    Example Request:
+    
+    ```
+    POST /experiment_trials
+    Host: <URL>:<PORT>
+    Content-Type: application/json
+    
+    {
+        "id" : "sha256sum of autotune object",
+        "url" : "http://localhost:8080/searchSpace",
+        "operation" : "EXP_TRIAL_GENERATE_NEW | EXP_TRIAL_GENERATE_SUBSEQUENT"
+    }
+    ```
+    
+    Response:
+    
+    | Status code | Response body |
+    | --- | --- |
+    | 200 | trial_number |
+    | 400 | 1 |
+    | 403 | 1 |
```

Comment:
Is it 1 or -1?

Reply:
It should be -1. Thanks for pointing it out. I have made the changes.